lib/application/printers/printers_actor/printers_actor_bloc.dart

```diff
@@ -75,21 +74,21 @@ class PrintersActorBloc extends Bloc<PrintersActorEvent, PrintersActorState> {
       linearProgressIndicator: const LinearProgressIndicator(),
     ).show(event.context);
 
-    final String printerIp = event.printer.deviceLastKnownIp.getOrCrash();
-    if (printerIp != null) {
-      launchUrl(
-        Uri.parse('http://$printerIp'),
-        mode: LaunchMode.externalApplication,
-      );
-    } else {
-      logger.e('Printer does not have lastKnownIp');
+    // final String printerIp = event.printer.deviceLastKnownIp.getOrCrash();
+    // if (printerIp != null) {
+    //   launchUrl(
+    //     Uri.parse('http://$printerIp'),
+    //     mode: LaunchMode.externalApplication,
+    //   );
+    // } else {
```

Comment:
You can't just comment code if it has warnings.

You have distorted a feature that easily like that

Reply:
the variable was not-nullable. So I didn't see a point of null check.